test/helpers/integration.rb

```diff
@@ -46,6 +46,10 @@ def teardown
 
   private
 
+  def silent_and_checked_system_command(*args)
+    assert(system(*args, out: File::NULL, err: File::NULL))
```

Comment:
Removed err as bundler keeps on complaining my version is not the same as the one in the `Gemfile.lock`
I also added an assert to make sure the command exited with success